site/content/docs/Repositories/_index.md

```diff
@@ -0,0 +1,213 @@
+
+* [Google Cloud Storage](#google-cloud-storage)
+* [Azure](#azure)
+* [Amazon S3](#amazon-s3) (and compatible)
+* [B2](#b2)
+* [Sftp](#sftp)
+* [Webdav](#webdav)
+* [Local storage](#local-storage)
+* [Http/s](#https-server)
+
+## Google Cloud Storage
+
+Google Cloud Storage is a globally unified, scalable, and highly durable object storage for developers and enterprises.
+
+### Creating a repository
+
+To create a repository in Google Cloud Storage you need to provision a storage bucket and install local credentials that can access that bucket. To do so:
+
+1. Create a storage bucket in [Google Cloud Console](https://console.cloud.google.com/storage/)
+2. Install [Google Cloud SDK](https://cloud.google.com/sdk/)
+3. Log in with credentials that have permissions to the bucket.
+
+```shell
+$ gcloud auth application-default login
+```
+
+After these preparations we can create Kopia repository (assuming bucket named `kopia-test-123`):
+
+```shell
+$ kopia repository create google --bucket kopia-test-123
+```
+
+At this point we should be able to confirm that Kopia has created the skeleton of the repository with 3
+files in it:
+
+```shell
+$ gsutil ls gs://kopia-test-123
+gs://kopia-test-123/kopia.repository
+gs://kopia-test-123/n417ffc2adc8dbe93f1814eda3ba8a07c
+gs://kopia-test-123/p78e034ac8b891168df97f9897d7ec316
+```
+
+### Connecting To Repository
+
+To connect to a repository that already exists, simply use `kopia repository connect` instead of `kopia repository create`. You can connect as many computers as you like to any repository, even simultaneously.
+
+```shell
+$ kopia repository connect google --bucket kopia-test-123
+```
+
+[Detailed informations and settings](/docs/reference/command-line/common/repository-connect-filesystem/)
+
+---
+
+## Amazon S3
+
+Kopia can connect to S3 compatible storage, such as [Amazon S3](https://aws.amazon.com/s3/), [minio.io](https://minio.io/), [Wasabi](https://wasabi.com/)
+
+### Creating a repository
+
+You will need your S3 bucket name, access key and secret access key.
+
+```shell
+$ kopia repository create s3 \
+        --bucket=... \
+        --access-key=... \
+        --secret-access-key=...
+```
+
+### Connecting To Repository
+
+```shell
+$ kopia repository connect s3 
+```
+
+[Detailed informations and settings](/docs/reference/command-line/common/repository-create-s3/)
+
+---
+
+## B2
+
+[B2](https://www.backblaze.com/b2/cloud-storage.html) is a very cheap yet fully featured S3 compatible storage.
+
+### Creating a repository
+
+You will need your B2 bucket name, key-id and key.
+
+```shell
+$ kopia repository connect b2 \
+        --bucket=... \
+        --key-id=... \
+        --key=...
+```
+
+### Connecting To Repository
+
+```shell
+$ kopia repository connect b2 
+```
+
+[Detailed informations and settings](/docs/reference/command-line/common/repository-connect-b2/)
+
+---
+
+## Sftp
+
+
+---
+
+## Webdav
+
```

Comment:
I pushed my modifications before I finished to write all examples. Just in case I wasn't going in the right direction, I didn't wanted to write all for nothing :). Ill add missing parts

Reply:
thanks!

Reply:
@SamuelBenard We can push this PR with the changes so far and you can create a new PR once you have the content with the examples for the other repository types.